test-case/multiple-pause-resume.sh

```diff
@@ -139,7 +139,8 @@ exit 1
 END
 }
 
-max_wait_time=$[ 10 * $repeat_count ]
+# maximum 5 second per individual repeat
+max_wait_time=$[ 5 * $repeat_count ]
```

Comment:
is this change related to the fix below? Seems not, so probably need to push the fix below to a quick PR, and the rest as improvements?

Reply:
I agree, I will make a separate PR for improvement